docs/source/user-guide/sql/data_types.md

```diff
@@ -30,11 +30,11 @@ the `arrow_typeof` function. For example:
 
 ```sql
 select arrow_typeof(interval '1 month');
-+-------------------------------------+
-| arrowtypeof(IntervalYearMonth("1")) |
-+-------------------------------------+
-| Interval(YearMonth)                 |
-+-------------------------------------+
++---------------------------------------------------------------------+
+| arrow_typeof(IntervalMonthDayNano("79228162514264337593543950336")) |
```

Comment:
why isn't it 1? 🤔

Reply:
I think it returns the nano second as the result? 
If it returns IntervalYearMonth the result will be 1, but now Interval will always return `IntervalMonthDayNano`.
See #5801